eth/filters/filter.go

```diff
@@ -128,7 +128,9 @@ func (f *Filter) Logs(ctx context.Context) ([]*types.Log, error) {
 		if header == nil {
 			return nil, errors.New("unknown block")
 		}
-		return f.blockLogs(ctx, header, false)
+		size, sections := f.sys.backend.BloomStatus()
+		lastIndexed := size * sections
+		return f.blockLogs(ctx, header, false, lastIndexed > header.Number.Uint64())
```

Comment:
unsure that it makes a difference, is `>` or `>=` correct here?

Reply:
Elsewhere, it is >

Reply:
![image](https://user-images.githubusercontent.com/13023275/205153056-198307ff-e095-4ed9-8bf8-e5005da1bbff.png)

Reply:
I thought about creating a `LastIndexed` function but wanted to tread lightly. idk.